storage_service/locations/models/duracloud.py

```diff
@@ -321,7 +321,7 @@ def _process_chunk(self, f, chunk_path):
 
         with open(chunk_path, "w") as fchunk:
```

Comment:
@jraddaoui, you may want to try opening it with modes `wb`. In binary mode, you should be able to write the byte sequence without having to do the conversion to unicode.

Reply:
Thanks @sevein!